packages/web/components/CurateTechnology/History/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import DOMPurify from 'isomorphic-dompurify';
+import EmptyScreen from '../../EmptyScreen';
+import { useTechnology } from '../../../hooks';
+import { stringToLocaleDate } from '../../../utils/helper';
+import { Container, ContentBox } from '../styles';
+import * as S from './styles';
+
+const History = () => {
+	const { technology } = useTechnology();
+
+	const comments = [...technology.technologyRevisionsHistory].reverse();
+
+	return (
+		<Container>
+			<ContentBox flexBasis="100%">
+				{comments.length ? (
+					comments.map((comment) => (
+						<S.Comment key={comment.id}>
+							<S.CommentTitle>
+								<p>
+									{comment.reviewer
+										? 'Comentários do curador'
+										: 'Comentários do pesquisador'}
+								</p>
+							</S.CommentTitle>
+
+							<S.CommentContent>
+								<span>
+									{stringToLocaleDate(comment.created_at, {
+										day: 'numeric',
+										month: 'long',
+										year: 'numeric',
+										hour: '2-digit',
+										minute: '2-digit',
+									})}
```

Comment:
Do you think we should use `useMemo` hook here?

Reply:
IMHO there's no gain by doing that. On the other hand we can harm performance